frontend/app/src/main/java/com/example/speechbuddy/compose/texttospeech/TextToSpeechScreen.kt

```diff
@@ -0,0 +1,131 @@
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center,
+        ) {
+            TitleUi(
+                title = stringResource(id = R.string.tts_text),
+                description = stringResource(id = R.string.tts_explain)
+            )
+
+            Spacer(modifier = Modifier.height(20.dp))
+
+            OutlinedTextField(
+                value = viewModel.textInput,
+                onValueChange = {
+                    viewModel.setText(it)
+                },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .verticalScroll(rememberScrollState())
+                    .height(300.dp),
+                textStyle = MaterialTheme.typography.bodyMedium,
+                shape = RoundedCornerShape(10.dp),
+                colors = TextFieldDefaults.outlinedTextFieldColors(
+                    focusedBorderColor = MaterialTheme.colorScheme.tertiary,
+                    unfocusedBorderColor = MaterialTheme.colorScheme.outline
+                )
+            )
+
+            Spacer(modifier = Modifier.height(20.dp))
+
+            // show PLAY/STOP depending on tts speaking
+            if (uiState.isButtonEnabled == ButtonStatusType.PLAY) {
+                Button(
+                    onClick = {
+                        viewModel.ttsStart(context)
+                    },
+                    colors = ButtonDefaults.buttonColors(
+                        containerColor = Color.Transparent,
+                        contentColor = MaterialTheme.colorScheme.onBackground
+                    )
+                ) {
+                    Text(
+                        style = MaterialTheme.typography.headlineMedium,
+                        text = stringResource(id = R.string.play_text)
+                    )
+                    Icon(
+                        Icons.Filled.PlayArrow,
+                        contentDescription = stringResource(id = R.string.play_text),
+                        modifier = Modifier.size(36.dp)
+                    )
+                }
+            } else {
+                Button(
+                    onClick = {
+                        viewModel.ttsStop()
+                    },
+                    colors = ButtonDefaults.buttonColors(
+                        containerColor = Color.Transparent,
+                        contentColor = MaterialTheme.colorScheme.onBackground
+                    )
+                ) {
+                    Text(
+                        style = MaterialTheme.typography.headlineMedium,
+                        text = stringResource(id = R.string.stop_text)
+                    )
+                    Icon(
+                        painterResource(R.drawable.stop_icon),
+                        contentDescription = stringResource(id = R.string.pause_text),
+                        modifier = Modifier.size(36.dp)
+                    )
+                }
+            }
```

Comment:
```suggestion
            TextToSpeechButton(
                buttonStatus = uiState.buttonStatus,
                onPlay = { viewModel.ttsStart(context) },
                onStop = { viewModel.ttsStop() }
            )
```

이 부분 너무 코드가 길어서 따로 빼면 좋을 것 같아요!